src/utils/yarn.js

```diff
@@ -2,7 +2,17 @@
 import type Package from '../Package';
 import * as processes from './processes';
 
-export async function run(pkg: Package, script: string, args: Array<string> = []) {
+export async function add(pkg: Package, args: Array<string> = []) {
```

Comment:
I'd prefer if this method accepted specific options instead of `args` like:

```js
add(pkg, {
  name: 'dep-1',
  version?: '^1.2.3',
  type: 'devDependencies'
});
```

- The `yarn.add` util should be converting JS options into Yarn CLI commands
- The `pyarn add` command should be converting pyarn CLI command into JS options